docs/getting-started/quick-start.md

```diff
@@ -22,6 +22,9 @@ docker-compose up
 > [!NOTE]
 > Replace `/dev/ttyACM0` with your serial device
 
+> [!NOTE]
+> Replace `-p 3000:3000` with your Server Port setting for zwave-js websocket server
```

Comment:
Same here

Reply:
Should be changed.  If I did it correctly